test/gas.js

```diff
@@ -272,5 +272,92 @@ describe("Gas", function() {
       let gasPrice = '0x2000'
       return testGasExpenseIsCorrect(gasPrice, false, new Web3(Ganache.provider({ mnemonic, gasPrice })))
     })
-  })
+    it('should calculate cumalativeGas and gasUsed correctly when multiple transactions are in a block', function (done){
+      let tempWeb3 = new Web3(Ganache.provider({
+        blockTime: .5, // seconds
+        mnemonic: "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
```

Comment:
This appears to be the same mnemonic as the one defined in the `mnemonic` variable above. Don't see why we can't just use that variable here, too.